beacon_node/src/cli.rs

```diff
@@ -1590,5 +1590,27 @@ pub fn cli_app() -> Command {
                 .action(ArgAction::Set)
                 .display_order(0)
         )
+        .arg(
+            Arg::new("delay-block-publishing")
+                .long("delay-block-publishing")
+                .value_name("SECONDS")
+                .action(ArgAction::Set)
+                .help_heading(FLAG_HEADER)
+                .help("TESTING ONLY: Artificially delay block publishing by the specified number of seconds. \
+                       DO NOT USE IN PRODUCTION.")
+                .hide(true)
+                .display_order(0)
+        )
+        .arg(
+            Arg::new("delay-data-column-publishing")
+                .long("delay-data-column-publishing")
+                .value_name("SECONDS") 
+                .action(ArgAction::Set)
+                .help_heading(FLAG_HEADER)
+                .help("TESTING ONLY: Artificially delay data column publishing by the specified number of seconds. \
+                       DO NOT USE IN PRODUCTION.")
+                .hide(true)
+                .display_order(0)
+        )
```

Comment:
The call to `publish_block` is triggered by the validator client calling `/eth/v2/beacon/blocks` or similar endpoint. Currently this endpoint has a default timeout value of:

[`slot_duration / HTTP_PROPOSAL_TIMEOUT_QUOTIENT`](https://github.com/sigp/lighthouse/blob/v6.0.1/validator_client/src/lib.rs#L316) = `6 seconds`

So there's an implicit requirement that the sum of these two timeouts not exceed that if you don't want to see error messages on the validator client.